SAKPaaS/src/app/core/services/location-provider.service.ts

```diff
@@ -60,4 +60,10 @@ export class LocationProviderService {
   public getLoadingLocationsState(): Observable<boolean> {
     return this.isLoadingLocations;
   }
+
+  public getDistanceToLocation(location: Location): number {
+    if (this.lastUpdatedPosition !== null) {
```

Comment:
kannst du da noch n return undefined oder null einbauen? wenn `this.lastUpdatedPostion` `null` ist, gibt er nichts zurück

Reply:
Oh ja lol😂 warum funktioniert das überhaupt😂

Reply:
Resolved in https://github.com/SAKPaaS/frontend/pull/135

Reply:
Hmm, nächstes mal dann auch in den branch bitte fixen xD aber egaaaaal